React/Fabric/Mounting/ComponentViews/SafeAreaView/RCTSafeAreaViewComponentView.mm

```diff
@@ -59,16 +59,19 @@ - (void)_updateStateIfNecessary
   insets.top = RCTRoundPixelValue(insets.top);
   insets.right = RCTRoundPixelValue(insets.right);
   insets.bottom = RCTRoundPixelValue(insets.bottom);
+
+  auto newPadding = RCTEdgeInsetsFromUIEdgeInsets(insets);
+  auto threshold = 1.0 / RCTScreenScale() + 0.01; // Size of a pixel plus some small threshold.
```

Comment:
These are these are outside the macOS diff and seemingly unused?

Reply:
I put those there to keep the iOS version unchanged. But we could apply the same change to both. The pointScaleFactor on the layout metrics for iOS is set to RCTScreenScale() anyways.

Reply:
Oh I see now. I would add a `// [macOS]` tag still do the changed iOS code. This seems like a candidate for "Simple upstream able refactor that can be done with almost no change for iOS, but makes macOS slightly easier" if you're up for that too :)

Reply:
Done